quill/common/src/components.rs

```diff
@@ -109,3 +109,13 @@ bincode_component_impl!(CreativeFlying);
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct Sneaking(pub bool);
 bincode_component_impl!(Sneaking);
+
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+/// A component on players that tracks if they are sprinting or not.
```

Comment:
I personally think comments should be above derive, but I don't judge.

Reply:
I am blind, they really should be. How do I unapprove

Reply:
Reloacated

Trying to look into the other change request but it might take me a bit because I'm new to Rust and ECS